src/station_cmd.cpp

```diff
@@ -5163,6 +5332,28 @@ uint FlowStatMap::GetFlowFromVia(StationID from, StationID via) const
 	return i->second.GetShare(via);
 }
 
+static CommandCost CheckBuildAbove_Station(TileIndex tile, DoCommandFlags flags, Axis, int height)
```

Comment:
Missing doxygen.

Reply:
Not documenting this one because we don't generally document callback implementations, but I have now documented the callback function signature instead.